src/panels/lovelace/editor/config-elements/config-elements-style.ts

```diff
@@ -25,5 +25,6 @@ export const configElementStyle = css`
   ha-textfield,
   ha-icon-picker {
     margin-top: 8px;
+    display: block;
```

Comment:
This is applying to every place that uses these styles. Did you verify that owrks?

Reply:
Yea. This is all good everywhere